README.md

```diff
@@ -119,34 +118,13 @@ dictionary MediaEffectInfo {
 partial dictionary VideoFrameMetadata {
   MediaEffectInfo backgroundBlur;
```

Comment:
As a followup, it may be useful to explain why we need this to be a dictionary instead of just an `EffectState` property.   For example, I would argue that it allows more metadata to be passed in the future for a specific effect without a breaking change, and give examples of that metadata (effect intensity, OS vs browser, etc.)